src/sites/doc/docs/taro/official-theme-react.en-US.md

```
@@ -0,0 +1,65 @@
+# Official theme
+
+## introduce
+
+NutUI provides multiple sets of official `UI` themes by default, while allowing new themes to be customized to a certain extent to meet the diverse visual needs of the business.
+
+####
+
+| theme description | scss file name |
+| --- | --- |
+| Jingdong APP theme (default) | `variables.scss` |
+| Jingdong B Mall Theme <a target="_blank" href="https://nutui.jd.com/h5/vue/4x/?jdb#/zh-CN/component/button" >Preview</a> | `variables-jdb.scss ` |
```

Markdownlint (0.37.0)

Spaces inside code span elements

(MD038, no-space-in-code)